modules/folder/main.tf

```diff
@@ -48,3 +62,30 @@ resource "google_compute_firewall_policy_association" "default" {
   name              = var.firewall_policy.name
   firewall_policy   = var.firewall_policy.policy
 }
+
+resource "google_assured_workloads_workload" "folder" {
+  count                     = var.assured_workload_config == null ? 0 : 1
+  compliance_regime         = var.assured_workload_config.compliance_regime
+  display_name              = var.assured_workload_config.display_name
+  location                  = var.assured_workload_config.location
+  organization              = var.assured_workload_config.organization
+  enable_sovereign_controls = try(var.assured_workload_config.enable_sovereign_controls, null)
+  labels                    = try(var.assured_workload_config.labels, {})
+  partner                   = try(var.assured_workload_config.partner, null)
```

Comment:
I'd skip `try`s here as well, default for labels is already managed by variable definition.

Reply:
Done